snowpack/tsconfig.json

```diff
@@ -10,8 +10,8 @@
     "strict": true,
     "sourceMap": true,
     "noImplicitAny": false,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
```

Comment:
I’m open to changing these, but what is the reason (I like it because it keeps the codebase clean and serves as a good reminder to remove dependencies if they’re no longer needed)?

Reply:
bah, I keep changing these when I do local development and forgetting to remove before merging.  Will remove.

(I wonder if we could add these as a test/lint step instead since it does make commenting out code very tedius while working)

Reply:
I’d be down for like a `tsconfig.production.json` file that has more strict rules or something, if that’s helpful (but it would also give feedback later after pushing to CI)

Reply:
followed up in #1224